kmk/extensions/peg_oled_display.py

```diff
@@ -98,16 +97,17 @@ def renderOledTextLayer(self, layer):
                 y=25,
             )
         )
+        self._display.show(splash)
         gc.collect()
 
     def renderOledImgLayer(self, layer):
         splash = displayio.Group()
-        self._display.show(splash)
         odb = displayio.OnDiskBitmap(
             '/' + self.returnCurrectRenderText(layer, self._views[0])
         )
         image = displayio.TileGrid(odb, pixel_shader=odb.pixel_shader)
         splash.append(image)
+        self._display.show(splash)

```

Comment:
Moves `self._display.show(splash)` to after setting up the `displayio.Group()` so that content no longer flashes/flickers when changing display content based on current layer.